scripts/setup.js

```diff
@@ -36,10 +37,15 @@ document.addEventListener("keypress",function(){
       finder.on('file', (file) => {
         if (path.extname(file) == ".mp3") {
           state.allFiles.push(file)
+          if(!state.nowplaying.instance) // For first time nowplaying.instance not exists so we check for it
+          {
+            state.load(state.allFiles[0]) // load first song
+          }
         }
       })
 
       finder.on('end', async () => {
+        state.allFiles.sort()
```

Comment:
`allFiles` is a list of file paths and it's being sorted by file names when it is supposed be based on the song name.

Reply:
@sanket143 if we sort by path then songs were sorted by name automatically.

Reply:
they won't get sorted by song names, they'll get sorted by filenames or their full path to be precise.

Reply:
@sanket143 made changes which are requested by you